SharpAdbClient/AdbClient.cs

```diff
@@ -344,33 +344,23 @@ public IEnumerable<ForwardData> ListForward(DeviceData device)
         }
 
         /// <inheritdoc/>
-        public async Task<Image> GetFrameBufferAsync(DeviceData device, CancellationToken cancellationToken)
+        public Framebuffer GetRefreshableFramebuffer(DeviceData device)
```

Comment:
@qmfrederik would it not be better to call the method CreateRefreshableFramebuffer?